ignite/handlers/lr_finder.py

```diff
@@ -302,12 +302,12 @@ def lr_suggestion(self) -> Any:
         if not self._history:
             raise RuntimeError("learning rate finder didn't run yet so lr_suggestion can't be returned")
         loss = self._history["loss"]
-        min_loss_idx = torch.tensor(loss).argmin()
+        min_loss_idx = torch.FloatTensor(loss).argmin()
```

Comment:
I think `torch.tensor` is the only correct way to create tensors. If you need to specify dtype as torch.float32:
```
torch.tensor(loss, dtype=torch.float32)
```
However, why do you propose this change ?

Reply:
Using `torch.FloatTensor` is a way to handle this warning 
```
UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
    min_loss_idx = torch.tensor(loss, dtype=torch.float32).argmin()
```